specification/databricks/resource-manager/Microsoft.Databricks/stable/2018-04-01/databricks.json

```diff
@@ -391,16 +391,20 @@
           "$ref": "#/definitions/ProvisioningState",
           "description": "The workspace provisioning state."
         },
-        "uiDefinitionUri": {
```

Comment:
remove property from stable version is breaking change, suggest to do it in new api version. Just to confirm, is this a Swagger file bug fixing, fixing Swagger file to align with existing service behavior? or new change?

Reply:
this is fixing swagger file to align with existing service behavior.

Reply:
@yungezz After discussing with the team internally, I have reverted this change.